boot/bootutil/src/encrypted.c

```diff
@@ -51,79 +43,49 @@
 
 #include "bootutil_priv.h"
 
-#if defined(MCUBOOT_ENCRYPT_KW)
-#if defined(MCUBOOT_USE_MBED_TLS)
-static int
-key_unwrap(const uint8_t *wrapped, uint8_t *enckey)
+#if defined(MCUBOOT_ENCRYPT_EC256) || defined(MCUBOOT_ENCRYPT_X25519)
+#if defined(_compare)
+static inline int bootutil_constant_time_compare(const uint8_t *a, const uint8_t *b, size_t size)
 {
-    mbedtls_nist_kw_context kw;
-    int rc;
-    size_t olen;
-
-    mbedtls_nist_kw_init(&kw);
+    return _compare(a, b, size);
+}
+#else
+static inline int bootutil_constant_time_compare(const uint8_t *a, const uint8_t *b, size_t size)
```

Comment:
Any reason this should be inline? Function call overhead is far from being a significant factor in execution time of mcuboot (nearly all time is spent in crypto primitives), and inline only serves to make the code larger.

Reply:
Agreed, I'll drop `inline` on all of these.